homeassistant/components/scene/velux.py

```diff
@@ -0,0 +1,59 @@
+"""
+Support for VELUX scenes.
+
+Connects to VELUX KLF 200 interface
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/scene.velux/
+"""
+import logging
+
+from homeassistant.components.scene import Scene
+
+
+DEPENDENCIES = ['velux']
+_LOGGER = logging.getLogger(__name__)
+DOMAIN = "velux"
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup velux scene plattform."""
+    if DOMAIN not in hass.data \
```

Comment:
Needs an update if `DATA_XY` is used. It looks like that the connection error is caught by the component already.